### DISCUSSION TITLE
Daquota?

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#158] [General]
Hello Renauld,

Nice to see some action on dlite again and the introduction of Daquota. What’s the plan behind Daquota?

Kind regards
Folkert

**Comment by renaudpawlak** [comment]
Very busy :) Plan is to launch ASAP (new project name is Daquota). Check https://www.daquota.io for the big picture. Daquota repo will provide Open Source apps and templates + Open Source components to plug in your apps. Still working on open sourcing the components. Still working on local-first sync and other UX issues with the IDE (getting better). Still working on finding some funding partners and business associates for the project, but already very promising leads.
